packages/supi/src/symlinkDependencyTo.ts

```diff
@@ -4,6 +4,13 @@ import symlinkDir = require('symlink-dir')
 
 export default function symlinkDependencyTo (alias: string, peripheralLocation: string, dest: string) {
   const linkPath = path.join(dest, alias)
+  // Don't symlink deps from packages that are outside of the monorepo root.
+  //   These may be symlinks from another project, and we don't want to break the other project.
+  // TODO(vjpr): Is `process.cwd()` a reliable way to get the monorepo root?
```

Comment:
to get the root of the repo, check `shrinkwrapDirectory`.

Reply:
Is there a singleton package I can use to get it, or I have to pass it into `symlinkDepTo()`?

Reply:
Pushed commit using shrinkwrapDirectory.